src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/analysers/JavaASTAnalyser.java

```diff
@@ -134,6 +135,9 @@ public class JavaASTAnalyser implements Analyser {
         JAVA_11_PARSER = JavaParserAdapter.of(new JavaParser(new ParserConfiguration()
             .setLanguageLevel(ParserConfiguration.LanguageLevel.JAVA_11)
             .setDetectOriginalLineSeparator(false)));
+
+        INDENT_LEVELS = new ArrayList<>(Arrays.asList("", makeNewWhitespace(4), makeNewWhitespace(8),
```

Comment:
We may not reach 24 very often or even 20. Should we lazily grow this and add to this cache when we create it the first time? Probably 4, 8, 12, 16 are more common and we can start there?

Reply:
It makes me nervous to hard code this. I would rather just compute it as necessary.

Reply:
I can remove this if we don't think it's worthwhile

Reply:
I would probably remove it